boards/arduino-nano-33-iot/Makefile.include

```diff
@@ -0,0 +1,8 @@
+PORT_LINUX ?= /dev/ttyACM0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
+
+# setup serial terminal
+include $(RIOTMAKE)/tools/serial.inc.mk
+
+# Include all definitions for flashing with bossa over USB
+include $(RIOTBOARD)/common/samd21-arduino-bootloader/Makefile.include
```

Comment:
```suggestion
# Include all definitions for flashing with bossa other USB
include $(RIOTBOARD)/common/samd21-arduino-bootloader/Makefile.include
# Include handling of serial and non-bossa programmers (if selected by user)
include $(RIOTMAKE)/boards/sam0.inc.mk
```

Reply:
The current state of this PR is exactly like the state of arduino-mkr in master. So I think it's ok.

Reply:
Ah yea sorry I just remembered that this part was recently refactored so it was my first suspect.
But the timer configuration got refactored too, just a longer time ago.